lua/lazyvim/plugins/extras/lang/rust.lua

```diff
@@ -42,7 +42,7 @@ return {
 
   {
     "mrcjkb/rustaceanvim",
-    version = "^4", -- Recommended
+    version = "^5", -- Recommended
```

Comment:
It's possible that rustaceanvim was pinned to 4.x because 5.x requires neovim>=0.10.0, while LazyVim currently requires only neovim>=0.9.0. It's probably better to keep neovim<0.10.0 on 4.x, using something like

```lua
version = vim.fn.has("nvim-0.10.0") == 1 and "^5" or "^4"
```

Reply:
I believe it should be better in the check for 0.10 to define the version as `false` to always update to latest HEAD.

Reply:
In that case the condition would become
```lua
version = vim.fn.has("nvim-0.10.0") == 0 and "^4" or false
```
I agree that for neovim>=0.10 rustaceanvim shouldn't be pinned to a specific major version.

Reply:
Done.